source/_integrations/thread.markdown

```diff
@@ -14,30 +14,158 @@ ha_platforms:
+The Thread configuration page shows three vendor-specific Thread networks.
+</p>
+
+These are all separate networks using different credentials. This means devices can't roam between the Thread networks.
+
+The <img width="30px" src='/images/integrations/thread/information-outline.png'> icon indicates that Home Assistant has the credentials for that network. In this case, only the credentials of the `home assistant` network are known.
+
+Home Assistant discovers all Thread border routers in your network because they send mDNS/DNS-SD announcements. These local announcements don't contain the network credentials. That's why you see the network there, but not the credentials.
+
+#### About the preferred network
+
+The intention of the **Preferred network** in Home Assistant is that it will be used as the default network when adding Thread⁻based devices.
+
+<div class="note">
+
+The **preferred network** function isn't completely implemented yet. In particular, when adding Matter devices through the companion apps, the preferred network of the mobile device is being used.
+
+</div>
+
+#### Making a network your preferred network
+
+You can only set a Thread network as preferred if the credentials are known.
+
+1. To import Thread credentials, you need your Android and iOS companion app.
+2. On your companion app, navigate to the Thread configuration page.
+   - You should see an **Import credentials** button on the lower right corner.
+
+   <img width="400" src='/images/integrations/thread/thread-import-credentials.png'>
+
+   - Importing the credentials allows a Google- or Apple-created Thread network to be the preferred network of Home Assistant.
+   <img width="400" src='/images/integrations/thread/thread-preferred-network.png'>
+
+#### Using Home Assistant as a <abbr title="Thread border routers">TBRs</abbr> without setting up another vendor's TBR first
+
+This scenario currently only works in one particular case, under the following conditions:
+
+- You do not have a Google <abbr title="Thread border routers">TBRs</abbr> set up yet (otherwise the Google <abbr title="Thread border routers">TBRs</abbr> will be the preferred BR, and Android APIs do not allow to change that currently)
+- You have the OpenThread Border Router add-on installed (or the Silicon Labs Multiprotocol add-on, which includes an OpenThread Border Router). For instructions on how to enable Thread on SkyConnect or Yellow, refer to the following guides:
+  - [Enabling Thread on SkyConnect](https://skyconnect.home-assistant.io/procedures/enable-thread/)
+  - [Enabling Thread on Yellow](https://yellow.home-assistant.io/procedures/enable-thread/)
+- You have an Android phone and the Home Assistant Companion App.
+- The devices you want to add to the network support Matter. For instructions on how to add Thread-based Matter devices, refer to the section [Adding a Matter device to Home Assistant](/integrations/matter/#adding-a-matter-device-to-home-assistant)
```

Comment:
There is a step missing:  Make sure that you've set Home Assistant as the preferred network first (step above).

Reply:
Or better stated: 

> Make sure that there are no other Thread networks present (see above).